tests/cloud-init

```diff
@@ -56,5 +56,31 @@ rm ./profile-key ./profile-key.pub ./additional-key ./additional-key.pub
 lxc profile unset default cloud-init.user-data
 lxc delete -f c1
 
+echo "==> Use the NoCloud datasource by using a disk device with a \"cloud-init:config\" source"
+lxc init ubuntu:n vm1 --vm -c cloud-init.user-data="$(cat <<EOF
```

Comment:
Could the VM be named `v1` please for consistency with other tests and instances? I find it easier to track visually as it better distinguish from the `--vm` flag.

Reply:
Please use `${IMAGE}` here too.